Dockerfile

```diff
@@ -7,7 +7,8 @@ ARG NEXT_TELEMETRY_DISABLED=1
 COPY . .
 
 RUN yarn install --frozen-lockfile
+RUN yarn build
 
-CMD ["yarn", "dev"]
+CMD ["yarn", "start"]
```

Comment:
This one doesn't matter in Kubernetes. We're overriding CMD in Dockerfile with "command" in k8s deployment manifest.

Reply:
https://github.com/fjelltopp/fjelltopp-infrastructure/blob/8156456b4344aaf2ec2111bc521a544c76f43a3c/roles/navigator/templates/kubernetes/navigator_ui.yaml#L89

Reply:
I agree, however I think that having a working meaningful default in the Dockerfile makes sense. That's what I've adviced Manoj to do.

Reply:
yeah, but it makes more sense to have local Docker running in dev mode, if anyone needs it, where Kubernetes doesn't care about it. I just mean that this change is not production related.